packages/components/src/icon/test/index.js

```diff
@@ -63,7 +63,7 @@ describe( 'Icon', () => {
 		expect( wrapper.prop( 'height' ) ).toBe( 24 );
 	} );
 
-	it( 'renders an svg element and passes the size as its width and height', () => {
+	it( 'renders an svg element and override its width and height', () => {
```

Comment:
This test is originally intended to test that the width/height of the svg element is respected, but should be overridden by the `size` prop of the `Icon` component.